Cmd bug fix

ear7h opened the pull request:
Fix `snapshot bounds ...` command

ear7h committed:
fixed bounds bug

ear7h committed:
added tests for snapshottile func

ear7h committed:
Merge branch 'master' into cmd-bug-fix

ARolek commented on cmd/snapshot/main.go:
```diff
@@ -63,15 +63,18 @@ func parseBounds(boundString string) {
+	x2, err := strconv.ParseFloat(strings.TrimSpace(bounds[3]), 64)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error: invalid bounds provided — %v\n", boundString)
+		//fmt.Fprintf(os.Stderr, "Error: unabled to parse %v(%v) as a float.\n", )
```
remove commented out code.

gdey replied:
You are ignoring all the errors for y1,x1, and y2. Here you need to duplicate the err check after each one.  If you are want to do it this way instead of using the for loop.

gdey commented on snapshotter_test.go:
```diff
@@ -12,6 +12,25 @@ import (
+
+	return f
+}
+
```
I don't like having test helper functions take in a testing.T. Let the test decide what it should do if the helper produces an error.

gdey committed:
Minor error fixes, and style changes.

We were ignoring the error value of parsing bound values.
Updated testing structure.
